### ISSUE TITLE
Headless + Backend install is missing templates

### ISSUE THREAD

**Issue by fauno** [spree/spree#10195]
## Context

Hi, I was just trying out the headless install on a new Rails project following the README.md, so I added these gems to the Gemfile and ran the generators.

```ruby
gem 'spree_api', '~> 4.1'                                                                                                                                    
gem 'spree_backend', '~> 4.1'                                                                                                                                
gem 'spree_auth_devise', '~> 4.1'                                                                                                                            
gem 'spree_gateway', '~> 3.7'
```

```bash
bundle exec rails g spree:install --user_class=Spree::User
bundle exec rails g spree:backend:copy_views
```

But when I try to open http://localhost:3000/admin I get redirected to `/login` and then I get a `Missing template spree/layouts/spree_application` exception.  I see this file exists in spree_frontend, but shouldn't it be included on spree_backend?

## Your Environment

```
Missing template spree/layouts/spree_application with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :jbuilder, :rabl]}. Searched in: * "/home/fauno/Projects/sutty/tienda.sutty.nl/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/spree_gateway-3.7.5/lib/views/backend" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/spree_auth_devise-4.1.0/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/spree_auth_devise-4.1.0/lib/views/backend" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/devise-4.7.1/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/spree_backend-4.1.6/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/doorkeeper-5.4.0/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/spree_api-4.1.6/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/spree_core-4.1.6/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/kaminari-core-1.1.1/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/actionmailbox-6.0.3.1/app/views" * "/home/fauno/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/actiontext-6.0.3.1/app/views" 
```

**Comment by fauno** [comment]
Visiting http://localhost:3000/admin/zones redirected to http://localhost:3000/admin/login and it works :)

**Comment by jorgearimitsu** [comment]
I'm having the same problem...

**Comment by damianlegawiec** [comment]
This was fixed in 4.2 release :)
